tests/Integration/WebTestCase.php

```diff
@@ -115,12 +115,12 @@ private static function runBeforeClassTraits()
         }
     }
 
-    protected function createClient(): Client
+    final protected function createClient(): Client
```

Comment:
Hardly any of these `protected` methods are intended to be overridden by extending classes.